ds3-sdk/src/test/java/com/spectralogic/ds3client/helpers/FileObjectGetter_Test.java

```diff
@@ -77,4 +85,27 @@ public void testThatFileReportsAsRegularOnWindows() throws IOException, Interrup
 
         assertFalse(caughtException.get());
     }
+
+    @Test
+    public void testThatSymbolicLinksAreResolved() {
+        Assume.assumeTrue(Platform.isMac());
```

Comment:
we should cherry-pick this commit as well:
https://github.com/SpectraLogic/ds3_java_sdk/commit/e808e7cd69d41789340a03d936d4aae825155a90